src/toil_scripts/adam_pipeline/adam_preprocessing.py

```diff
@@ -36,11 +36,12 @@
 
 import yaml
 from toil.job import Job
```

Comment:
I'm not suggesting changing the order. I'm suggesting to move the line of whitespace from before `import yaml` to after `import yaml` so that `import yaml` is in the previous block of imports and the second block of imports is just imports `from toil.*`.